pkg/grpc/config/config.proto

```diff
@@ -94,7 +94,8 @@ message Route {
   // AuthorizationHeaderMode set_authorization_header = 54;
 
   bool preserve_host_header = 24;
-  bool pass_identity_headers = 25;
+  bool deprecated_pass_identity_headers = 25;
```

Comment:
What's the history behind the existing pass_identity_headers field?

Is it backwards incompatible to change a proto field from required to optional?

Reply:
I wasn't sure if it was backwards compatible. Looking at how optional works, I think it should be ok to modify the existing field, so I will remove this one.